diff --git a/node_modules/@base-org/account/dist/core/constants.js b/node_modules/@base-org/account/dist/core/constants.js
index 3b50292..2ed8dba 100644
--- a/node_modules/@base-org/account/dist/core/constants.js
+++ b/node_modules/@base-org/account/dist/core/constants.js
@@ -1,4 +1,4 @@
-import pkg from '../../package.json' with { type: 'json' };
+import pkg from '../../package.json';
 export const CB_KEYS_URL = 'https://keys.coinbase.com/connect';
 export const CB_WALLET_RPC_URL = 'https://rpc.wallet.coinbase.com';
 export const PACKAGE_NAME = pkg.name;
